doc/tutorials/spark_estimator.rst

```diff
@@ -276,3 +276,11 @@ An example submit command is shown below with additional spark configurations an
 When rapids plugin is enabled, both of the JVM rapids plugin and the cuDF Python package
 are required. More configuration options can be found in the RAPIDS link above along with
 details on the plugin.
+
+Advanced Usage
+==============
+
+XGBoost needs to repartition to the num_workers to ensure there will be num_workers training
+tasks running at the same time, but repartition is a costly operation. To avoid repartition,
+users can set ``spark.sql.files.maxPartitionNum`` and ``spark.sql.files.minPartitionNum``
+to num_workers.
```

Comment:
Could you please elaborate on when a user might want to force reparittion dataset as well?